Gordon360/Models/CCT/Housing/Applicant.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("ApplicationID", "Applicant1")]
+[Table("Applicant", Schema = "Housing")]
+public partial class Applicant
```

Comment:
I'm not sure that I understand the point of this table as it stands. Is it not a duplicate of `[dbo].[Housing_Applicants]`? Are you reworking the existing structure or creating another structure to do the same thing?

Reply:
Sorry for the confusion! I think these two tables are not duplicate. `[Housing_Applicants]` was the table created by the "Apartment Application" team, and this table is created by us as the "Housing Lottery" team. They have different uses.